### PR TITLE
Remove unused service in UserService

### PR DESCRIPTION
**Pull request by marcospri**

### CODE REVIEW

File: lms/services/user.py [hunk 1]
```diff
@@ -84,7 +83,6 @@ def factory(_context, request):
     """Service factory for the UserService."""
 
     return UserService(
-        request.find_service(name="application_instance"),
         request.db,
         request.registry.settings["h_authority"],
     )
```
Comment (jon-betts) [inline lms/services/user.py:88]:
```suggestion
    return UserService(request.db, request.registry.settings["h_authority"])
```

I find it's always worth knocking off the trailing commas to see if `black` will back things up onto a single line.

File: tests/unit/lms/services/user_test.py [hunk 2]
```diff
@@ -56,20 +56,18 @@ def user(self, lti_user, application_instance):
         )
 
     @pytest.fixture
-    def service(self, db_session, application_instance_service):
+    def service(self, db_session):
         return UserService(
-            application_instance_service,
             db_session,
             h_authority="authority.example.com",
         )
```
Comment (jon-betts) [inline tests/unit/lms/services/user_test.py:63]:
```suggestion
    return UserService(db_session, h_authority="authority.example.com")
```

File: tests/unit/lms/services/user_test.py [hunk 3]
```diff
@@ -56,20 +56,18 @@ def user(self, lti_user, application_instance):
...
 class TestFactory:
-    def test_it(self, pyramid_request, application_instance_service, UserService):
+    def test_it(self, pyramid_request, UserService):
         user_service = factory(sentinel.context, pyramid_request)
 
         UserService.assert_called_once_with(
-            application_instance_service,
             pyramid_request.db,
             pyramid_request.registry.settings["h_authority"],
         )
```
Comment (jon-betts) [inline tests/unit/lms/services/user_test.py:73]:
```suggestion
        UserService.assert_called_once_with(
            pyramid_request.db, pyramid_request.registry.settings["h_authority"]
        )
```

**Review by jon-betts — APPROVED** [review]
Yep, good catch. 

When things get shorter / look short enough I like to try knocking trailing commas off to see if `black` will reformat things shorter. I've added some suggestions.

File: lms/services/user.py (hunk 1 above)
Comment (marcospri) [reply lms/services/user.py]: Github didn't allow me to commit these from here but just pushed them :+1:

**Pull request merged**